src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs

```diff
@@ -115,6 +114,8 @@ protected bool NeedsGeneratedNullableAttribute
             {
                 _embeddedTypesManagerOpt = new NoPia.EmbeddedTypesManager(this);
             }
+
+            _nonNullTypesAttribute = Compilation.GetWellKnownType(WellKnownType.System_Runtime_CompilerServices_NonNullTypesAttribute);
```

Comment:
>_nonNullTypesAttribute = Compilation.GetWellKnownType(WellKnownType.System_Runtime_CompilerServices_NonNullTypesAttribute); [](start = 12, length = 123)

It feels like we do not use consistent way to lookup the type. In one place we use ```SourceModule.LookupTopLevelMetadataType``` here we use GetWellKnownType. Consider using the same way to lookup the type in all places. Also, should we check whether this is indeed "injected" type?